containers/account/AccountDetails.tsx

```diff
@@ -74,6 +75,14 @@ export const AccountDetails: FC<{
     }
   }, [serviceClient, account])
 
+  const [copyNoteToggle, setCopyNoteToggle] = useState<boolean>(false)
+  const [copyNoteText, setCopyNoteText] = useState<string>("")
+  const copyNotification = (text: string) => {
+    setCopyNoteToggle(true)
+    setCopyNoteText(text)
+    setTimeout(() => setCopyNoteToggle(false), 5000)
```

Comment:
the setTimeout doesn't clear on a additional click, when you click an other copy button within 5 seconds the snackbar hides really fast